wordpress-mobile/WordPress-iOS
# Update notifications list heading style

Fixes #22464

## Description

This PR updates the notifications list section heading style.

| Before | After | After - List Scrolled |
| ------ | ----- | -------------------- |
| ![](https://github.com/wordpress-mobile/WordPress-iOS/assets/9609223/aefad86d-9a84-4419-b737-3abf6a6945e2) | ![](https://github.com/wordpress-mobile/WordPress-iOS/assets/9609223/d34584c7-d78f-40fb-917b-6c973509f56d)  | ![](https://github.com/wordpress-mobile/WordPress-iOS/assets/9609223/d1dcbb29-2865-4084-8d38-58433a0b526e) |

## Test Instructions

1. Run the Jetpack app and navigate to the notifications tab.
2. The UI of the heading (Today) should match the new design. (see: yWt5gg3nWORhu079Qfv3mS-fi-51_1125)
3. Switch to the dark/light theme to see the color works in two themes.

## Regression Notes
1. Potential unintended areas of impact
Notifications list section titles should be visible.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing.

3. What automated tests I added (or what prevented me from doing so)
None.

## PR submission checklist:

Testing checklist:
